backend/spec/lib_marcxml_bib_converter_spec.rb

```diff
@@ -335,7 +335,7 @@ def by_num(id)
         expect(@notes).to include('Resource--CustodialHistory-AT.')
       end
 
-      it "maps datafield[@tag='630'] to subject" do
+      it "maps datafield[@tag='630' or @tag='130' or @tag='730'] to subject" do
```

Comment:
Hi @ngeraci - this looks good, but this line is just changing the description of the test without adding those two fields to the test (which would require adding the `130` and `730` data fields to the sample document at the top, and changing the expected count below to `3`.

Reply:
Okay, thanks for clarifying. I was a little confused looking at the test and sample document, because it seemed like a number of fields that do get imported aren't in the sample document/test. So I wasn't sure of the criteria for what is included in the sample document.

Is it better to add these fields to the sample document/test, or to revert the description change in this line?

Reply:
I think it is slightly better to add the fields and help ensure this change doesn't get lost in the future. But reverting the description would be fine with me too.

Reply:
Ok -- after some trial and error, I'm going to revert the description.

I made an attempt to add the fields to the test, but it turned out to be a bit complex. Since duplicate subjects aren't imported, simply adding the `130` and `730` data fields to the sample document at the top, and changing the expected count below to `3` caused the test to fail. The actual count stayed stuck at 1, since the `130` and `730` field values were duplicate subjects that failed to import.

Looking at the test code, the MARCXML import/export map, and the subject spec, I'm not seeing an easy fix to have the test import all three fields as unique subject values. It seems like achieving that would need making more changes to how the test works & how other subject fields are tested.

Reply:
I see. - yes, it sounds like we'd need to add some more test records to test this correctly.